trivup/apps/OauthbearerOIDCApp.py

```diff
@@ -165,12 +197,6 @@ def generate_valid_token_for_client(self):
             self.send_error(400, 'Accept field should be "application/json"')
             return
 
-        if post_data is None:
```

Comment:
Why was this moved above the auth and accept checks?

Reply:
I moved it above just after we read the post_data. Do you think the pervious place is better?